subsys/bluetooth/host/conn.c

```diff
@@ -2003,30 +2003,30 @@ void bt_conn_notify_remote_info(struct bt_conn *conn)
 
 void bt_conn_notify_le_param_updated(struct bt_conn *conn)
 {
+	uint16_t interval = conn->le.interval_us / BT_HCI_LE_INTERVAL_UNIT_US;
```

Comment:
Should this be 
```suggestion
	uint16_t interval_1250us = conn->le.interval_us / BT_HCI_LE_INTERVAL_UNIT_US;
```

To follow the same naming used in the API fields?